src/library_glfw.js

```diff
@@ -1640,7 +1674,19 @@ var LibraryGLFW = {
 
   glfwGetClipboardString: (win) => {},
 
-  glfwMakeContextCurrent: (winid) => {},
+  glfwMakeContextCurrent: function(winid) {
+    if (winid !== null && GLFW.windows !== null) {
+      for (var i in GLFW.windows) {
+        if (GLFW.windows[i].id == winid) {
+          Module['ctx'] = Module['ctx' + (winid - 1)];
```

Comment:
Can we use `GLFW.contexts` here instead to putting more stuff on the Module object?

Reply:
Sure, additionally I'm gonna put the canvases into `GLFW.canvases`

Reply:
Done